integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwMiiDomain.java

```diff
@@ -140,17 +144,17 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
    */
   @Order(1)
   @Test
-  @DisplayName("Create FMW Domain model in image")
+  @DisplayName("Create JRF Domain model in image")
```

Comment:
Shouldn't this be FMW instead of JRF, just like other changes you made?

Reply:
Yes. Done